renovate.json

```diff
@@ -16,27 +16,37 @@
     "renovate"
   ],
   "commitMessageSuffix": "🌟",
-  "prHourlyLimit": 1,
-  "prConcurrentLimit": 1,
+  "prHourlyLimit": 2,
+  "prConcurrentLimit": 10,
   "updateNotScheduled": false,
   "timezone": "America/New_York",
   "schedule": [
-    "before 3am on the first day of the month"
+    "before 5am every weekday"
   ],
-  "dependencyDashboardApproval": true,
+  
   "packageRules": [
     {
       "matchPackagePatterns": [
-        "*"
+        "^electron"
       ],
-      "semanticCommitType": "chore"
+      "semanticCommitType": "chore",
```

Comment:
I think this will need to be updated to match our commit types for the changelog. I think this should be `dependency` based on the categories [here](https://github.com/cypress-io/cypress/blob/develop/scripts/semantic-commits/change-categories.js) and we'll need to disable the [commit scope](https://docs.renovatebot.com/semantic-commits/#changing-the-semantic-commit-scope).

Reply:
yeah agreed.  the default scope is obnoxious so I added the config to disable it